doc/requirements.txt

```diff
@@ -0,0 +1 @@
+sphinx
```

Comment:
We should not need to define sphinx within the requirements.txt. As far as I know any package defined within requirements.txt means they are needed to run the application. Sphinx is just needed for development when we need to build & release.

The sphinx package is already included in the test-requirements.txt. Which contains other packages for development.

```
coverage!=4.4,>=4.0
mock>=2.0.0
sphinx>=1.6.2
testtools>=2.2.0
flake8>=3.5.0
```
What do you think @seandst?

Reply:
Those are specifically docs requirements, which I added before creating a docs dependency on miqcli itself. At that point they became redundant, so the file should just go away. +1